database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+
```

Comment:
Add this option so it fails when something where fails. I see some logs in the CI that says that the kubectl wait timed out while waiting for condition:

```suggestion
set -o errexit
```

Reply:
Fixed.